etc/otel-jmx.config.yaml

```diff
@@ -12,25 +11,46 @@ rules:
        metric: kafka.streams.records.lag
        type: gauge
        desc: the current lag of the partition
-       unit: '{messages}'
+       unit: '{records}'
  - bean: kafka.consumer:type=consumer-fetch-manager-metrics,client-id=*
    metricAttribute:
      clientid: param(client-id)
```

Comment:
@rodesai should (or can) this be `client-id` instead of `clientid`?

Reply:
I suppose it could, but we'd need to change the policy as well. Let's do that in a follow up iteration.